test/e2e/catalog_e2e_test.go

```diff
@@ -272,6 +269,15 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 		})
 		Expect(err).ShouldNot(HaveOccurred())
 
+		var updatedConfigMap *corev1.ConfigMap
+		Eventually(func() types.UID {
+			var err error
+			// Get updated configmap
+			updatedConfigMap, err = c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
+			Expect(err).ShouldNot(HaveOccurred())
```

Comment:
I have a couple of comments/questions:

- Do we need to add a length check to the `ObjectMeta.OwnerReferences` to ensure there's at least a single owner reference to avoid a panic down the line? It's unknown to me whether this is overkill, and whether we can rely on the owner references always being present. I'm mainly interested in guarding against panics being produced and us not catching them and/or making them harder to debug in the future.
- nit: Can we handle the error explicitly vs. using the gomega assertions, e.g.

```go
updatedConfigMap, err := ...
if err != nil {
    return err
}
...
```

/lgtm

Reply:
Thanks!  I'll make both changes.